src/compo/tempo_nc2ioda.py

```diff
@@ -338,10 +359,14 @@ def main():
         "0 normal, 1 suspicious, 2 bad",
         type=int, default=0)
     optional.add_argument(
-        '-n', '--thin',
+        '-t', '--thin',
```

Comment:
just make a note but we have been removing the shorthand flags in our converters making the explicit `--thin` required to make them more obvious